embedded-aerospike/src/main/java/com/testcontainers/aerospike/AerospikeTestOperationsAutoConfiguration.java

```diff
@@ -21,11 +21,11 @@
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
  */
-package com.playtika.test.aerospike;
+package com.testcontainers.aerospike;
 
 import com.aerospike.client.AerospikeClient;
-import com.playtika.test.common.properties.InstallPackageProperties;
-import com.playtika.test.common.utils.PackageInstaller;
+import com.testcontainers.boot.common.properties.InstallPackageProperties;
```

Comment:
```com.testcontainers.boot``` not clear as testcontainers supports tons of languages and tools, please add spring keyword